lib/user-methods.js

```diff
@@ -62,6 +62,19 @@ function checkObjectTypes(obj, types) {
     }
 }
 
+function normalizeSyncUrl(authUrl, syncUrl) {
+    let parsedAuthUrl = new URL(authUrl);
```

Comment:
These all should be `const` (although I know that throughout the codebase we have been very liberal with the usage of `let`).